src/test/resources/application.conf

```diff
@@ -33,7 +33,7 @@ mysql {
   port = 3306
   port = ${?MYSQL_PORT}
   db {
-    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/testdb?createDatabaseIfNotExist=true&useSSL=false"
+    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/testdb?createDatabaseIfNotExist=true&useSSL=false&rewriteBatchedStatements=true"
```

Comment:
this is what we use in production, it allows the mysql jdbc driver to batch inserts into a single statement